### PR TITLE
Fix build error

### PR DESCRIPTION
**Pull request by ogapants**
Thank you for your good sample!
I encountered some build errors, but it was fixed.

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     android:layout_height="match_parent"
     tools:context="android.marshi.app.nestedwebview.sample.MainActivity">
 
-    <android.marshi.nestedwebview.NestedWevViewAppBarLayout
+    <android.marshi.nestedwebview.NestedWebViewAppBarLayout
```
Comment (ogapants) [inline app/src/main/res/layout/activity_main.xml:10]: fix typo

File: build.gradle [hunk 2]
```diff
@@ -7,7 +7,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.3.3'
-        classpath 'com.novoda:bintray-release:0.5.0'
+        classpath 'com.novoda:bintray-release:0.8.0'
```
Comment (ogapants) [inline build.gradle:10]: https://stackoverflow.com/questions/47470631/no-such-property-for-runtime-for-class-org-gradle-api-attributes-usage/49546085#49546085

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```
Comment (ogapants) [inline gradle/wrapper/gradle-wrapper.properties:3]: https://github.com/novoda/bintray-release/issues/213#issuecomment-395932354

File: gradlew [hunk 4]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (ogapants) [inline gradlew:1]: It's just a generated files

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (marshi) [reply app/src/main/res/layout/activity_main.xml:10]: 😂

**Comment by marshi** [comment]
Thank you for your contribution!!
Also I'm sorry for terrible typo.
I've confirmed it build successfully.

Looks Good To Me. Thank you!

**Pull request merged**